ui/src/components/CoreConcepts/Counter.js

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+import CounterDisplay from './CounterDisplay';
+import IncrementButton from './IncrementButton';
+import DecrementButton from './DecrementButton';
+import ResetButton from './ResetButton';
+
+function Counter () {
+    const [count, setCount] = useState(0);
+
+    function increment () {
```

Comment:
There is a naming convention for event handlers; it's recommended to name them like `handleSomething`; preferably, it should be about the actual event it supposes to handle. So here I would've called it `handleIncrementButtonClick.` The behavior and implementation of `increment` could change over time; for example, it would do something other than increment. The name `handleIncrementButtonClick` would always reflect what this function handles.

The same goes for `decrement.`

It's not a requirement from me, but a suggestion.

Reply:
Fixed!